crates/ide_completion/src/item.rs

```diff
@@ -120,16 +120,8 @@ impl fmt::Debug for CompletionItem {
     }
 }
 
-#[derive(Debug, Clone, Copy, Ord, PartialOrd, Eq, PartialEq)]
-pub enum CompletionScore {
-    /// If only type match
-    TypeMatch,
-    /// If type and name match
-    TypeAndNameMatch,
-}
-
 #[derive(Debug, Clone, Copy, Ord, PartialOrd, Eq, PartialEq, Default)]
-pub struct Relevance {
+pub struct CompletionRelevance {
```

Comment:
I renamed this to include the `Completion` prefix for consistency with the other items exported from `ide_completion`.

Reply:
>with the other items *exported* from

Nice, I never though about "prefix, if exporting" rule-of-thumb!

But yeah, it's not like there's a specific name scheme here.